src/AspNet.Security.OpenIdConnect.Server/project.json

```diff
@@ -21,8 +21,8 @@
         },
 
         "Microsoft.Framework.WebEncoders.Core": "1.0.0-*",
-
-        "Newtonsoft.Json": "7.0.1"
+        "Newtonsoft.Json": "7.0.1",
+        "System.Dynamic.Runtime": "4.0.0"
```

Comment:
visual studio was showing a "module should be referenced" around the JObjects error. I see now that it actually does build anyway.